### ISSUE TITLE
Podam doesn't support @NotEmpty and @NotBlank annotation from bean validation from Jakarta EE 8

### ISSUE THREAD

**Issue by mtedone** [mtedone/podam#92]
Adding a @NotNull annotation to a Pojo attribute causes Podam not to fill that value. Although it's impossible to guarantee that Podam will create a value for an attribute annotated with @NotNull, e.g. for non-instantiable types, when a field is annotated with @NotNull Podam should behave as if that field was not annotated, e.g. follow default behaviour, which is to fill the possible fields with values anyway. This behaviour should also be documented because users must know that if they annotate a non instantiable attribute with @NotNull, validation will fail.

**Comment by daivanov** [comment]
Fixed in Podam 5.4.0 or later.

**Comment by daivanov** [comment]
There should be no case when Podam won't produce a POJO for a request. In worst case user have to do it external factory, but there is no real excuse for not using @NotNull. If one will manage to find the case, this is a bug and it should be reported here.

**Comment by rvillablanca** [comment]
Hi, I was wondering if `@NotNull` is still being supported? For a pojo like:

```java
@Getter
@Setter
public class SamplePojo {

    @NotNull
    @NotEmpty
    private String name;

    @NotNull
    private Integer age;
}
```

Properties are not being populated, but I remove them it works fine

**Comment by daivanov** [comment]
Hi,

The thing here is @NotEmpty (and @NotBlank) were added in Jakarta EE 8, which was released later than the latest Podam version. It can be added to the next version.

Thanks, Daniil

**Comment by daivanov** [comment]
But this also means we have to drop Java 6 and Java 7 support...

Thanks, Daniil

**Comment by rvillablanca** [comment]
Thanks!

**Comment by daivanov** [comment]
Fixed in Podam 7.2.4 or later.
